inst/text/shiny/shiny_css.css

```
@@ -143,3 +92,109 @@ th, td {
 }
 
 /*
+
+
+/* CSS styles used for flexdashboard */
```

I think this is maybe misleading comment as it implies these styles are only used on flexdashboards when in fact they are flexdashboard-like styles being lifted for use elsewhere. 
```suggestion
/* CSS styles from flexdashboard */
```